README.md

```diff
@@ -1,2 +1,248 @@
-# web-devops-roadmap
-DevOps Roadmap for web developers
+# Web DevOps Roadmap
+
+This repository contains initial DevOps roadmap for all the web developers. Anyone can start this training with their existing learning projects. The approximate time for complete these practicals is 10 days.
+
+# Guidelines
+- Before starting any practice, it's essential to conduct research and learn the necessary concepts.
+
+- Create a GitHub/GitLab repository for each Practical if one doesn't already exist.
+
+- Learn Basic DevOps Principles:
```

Comment:
- Learn Basic DevOps Principles:
  - How to use Version Control effectively for your codebase as Git is Fundamental.
  - How to write unit tests.
  - Understand the core DevOps concepts like continuous integration, continuous deployment, and automation.
  - Basic scripts to get an overall understanding of scripting language.
  - Basic security practices like exposing env vars, Inbound/Outbound security rules, etc…